src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -15,10 +15,10 @@
     <tags>vstest visual-studio unittest testplatform mstest microsoft test testing</tags>
   </metadata>
   <files>
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.targets" target="runtimes\any\native" />
+    <file src="Microsoft.TestPlatform.Build\netstandard2.0\Microsoft.TestPlatform.targets" target="runtimes\any\native" />
 
     <!-- Add localized resources -->
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\**\*.dll" target="lib\netstandard1.3" />
+    <file src="Microsoft.TestPlatform.Build\netstandard2.0\**\*.dll" target="lib\netstandard2.0" />
```

Comment:
We need to check how is Tp.build package is inserted in CLI, if they are unzipping, & then copying from ns1.3 folder explicitly, this would fail now